docs/dtc-CurrentDTCRetriever.js

```
@@ -0,0 +1,153 @@
+    ecuAddress: null,
+    canID: null,
+    headerFormat: null,
+    protocolConfig: null,
+    protocolState: BaseDTCRetriever.PROTOCOL_STATES.INITIALIZED,
+  };
+
+  constructor(ecuDataRetriever = null) {
+    super(ecuDataRetriever);
+
+    if (CurrentDTCRetriever._instance) {
+      return CurrentDTCRetriever._instance;
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)